tests/serialization/test_pydantic_fields.py

```diff
@@ -27,3 +63,22 @@ class ConstrainedType(AvroBaseModel):
 
     with pytest.raises(ValidationError):
         ConstrainedType.deserialize(b'{"value": 0}', serialization_type="avro-json")
+
+
+def test_generic_class_type_serialize():
```

Comment:
Could you add a `parametrize` to test the `avro` serialization as well? `@pytest.mark.parametrize(
    "serialization_type", [AVRO, AVRO_JSON,]`

Reply:
Absolutely